build/pipelines/templates-v2/job-build-package-wpf.yml

```diff
@@ -27,6 +27,9 @@ parameters:
   - name: publishArtifacts
     type: boolean
     default: true
+  - name: signingIdentity

```

Comment:
heck at this rate, we should almost have a `do-codesign.yml` template that takes one of these `signingIdentity` objects we're passing around